website/docs/flags.md

```diff
@@ -0,0 +1,39 @@
+| **Listeners**              |                                                                           |                                 |                                        |
+| `listen-addr`          | FerretDB address for incoming TCP connections                             | `FERRETDB_LISTEN_ADDR`          | `"127.0.0.1:27017"`                    |
+| `listen-unix`          | FerretDB Unix domain socket path. If empty - Unix socket is disabled      | `FERRETDB_LISTEN_UNIX`          |                                        |
+| **Handlers**               |                                                                           |                                 |                                        |
+| `handler`              | FerretDB backend handler: 'dummy', 'pg', 'tigris'                         | `FERRETDB_HANDLER`              | `"pg"`                                 |
+| `postgresql-url`       | PostgreSQL URL for `pg` handler                                           | `FERRETDB_POSTGRESQL_URL`       | `"postgres://127.0.0.1:5432/ferretdb"` |
+| `tigris-url`           | Tigris URL for 'tigris' handler                                           | `FERRETDB_TIGRIS_URL`           | `"127.0.0.1:8081"`                     |
+| `tigris-client-id`     | [Tigris Client ID][tigris-docs-auth]                                      | `FERRETDB_TIGRIS_CLIENT_ID`     |                                        |
+| `tigris-client-secret` | [Tigris Client secret][tigris-docs-auth]                                  | `ferretdb_tigris_client_secret` |                                        |
+| `tigris-token`         | [Tigris token][tigris-docs-auth]                                          | `FERRETDB_TIGRIS_TOKEN`         |                                        |
+| **TLS**                    |                                                                           |                                 |                                        |
+| `listen-tls`           | See [Securing connections with TLS][securing-with-tls]                    | `FERRETDB_LISTEN_TLS`           |                                        |
+| `listen-tls-cert-file` | See [Securing connections with TLS][securing-with-tls]                    | `FERRETDB_LISTEN_TLS_CERT_FILE` |                                        |
+| `listen-tls-key-file`  | See [Securing connections with TLS][securing-with-tls]                    | `FERRETDB_LISTEN_TLS_KEY_FILE`  |                                        |
+| **Operation Modes**        |                                                                           |                                 |                                        |
+| `mode`                 | See [Operation modes](/operation_modes.md)                                | `FERRETDB_MODE`                 | `"normal"`                             |
```

Comment:
I think the link should be `./operation_modes.md`.

Reply:
Both of them are in root directory, so functionally it doesn't make any change, do we prefer relative paths?

Reply:
Did you try out the links? Running `task docs-dev` fails because of the links.

Reply:
Weird
```sh
 task docs-dev
task: [docs-dev] docker compose run --rm --service-ports docusaurus-docs start --host=0.0.0.0
[INFO] Starting the development server...
[SUCCESS] Docusaurus website is running at: http://localhost:3000/

✔ Client
  Compiled successfully in 8.65s

client (webpack 5.74.0) compiled successfully

✔ Client
  Compiled successfully in 135.23ms

client (webpack 5.74.0) compiled successfully
```

Reply:
Did you try `task env-down init`?

Reply:
Yea. I did. Really weird. But I guess it should work anyways.